web-registry/src/components/organisms/CreditBatches.tsx

```diff
@@ -25,6 +25,8 @@ interface BatchRowData {
   batch_denom: string;
   class_id: string;
   total_amount: number;
+  amount_active?: number; // calculated
```

Comment:
why not using "tradable" as in the response from the ledger?

Reply:
using now - and re-mapped the data to make it sortable by these supply fields